### ISSUE TITLE
Missing HelpDesk Center NavBar buttons

### ISSUE THREAD

**Issue by ehlersd** [zendesk/zendesk_sdk_ios#35]
Not sure what the problem is, when I use this line, the "Contact Us" right button doesn't display:
```
[ZDKHelpCenter showHelpCenterWithNavController:self.mainViewController.navigationController];
```

But when I use this line, the "Contact Us" right button is there, but the left back arrow doesn't display:
```
[ZDKHelpCenter presentHelpCenterWithNavController:self.mainViewController.navigationController];
```
Is there anyway to control this buttons?

**Comment by tecknut** [comment]
Hey Darren, 

Can you describe the issue with a bit more detail please?  `-showHelpCenterWithNavController:` is working as expected in our sample app. 

If you are looking to use `-presentHelpCenterWithNavController:` you could try setting the `self. mainViewController.navigationController.modalPresentationStyle` to the style you wish. This should give you a left cancel button. 

Thanks

**Comment by tecknut** [comment]
This is fixed in 1.4.1.1
